doc/releases/changelog-dev.md

```diff
@@ -54,6 +54,21 @@
 
   For more details on using pattern matching optimization you can check the corresponding documentation and also the
   following [paper](https://dl.acm.org/doi/full/10.1145/3498325).
+  
+* Added two new templates the `HilbertSchmidt` template and the `LocalHilbertSchmidt` template.
+  [(#2364)](https://github.com/PennyLaneAI/pennylane/pull/2364)
+  
+  ```python
+  from pennylane.templates import HilbertSchmidt
+
+  with qml.tape.QuantumTape(do_queue=False) as u_tape:
+      qml.Hadamard(wires=0)
+
+  def v_function(params):
+      qml.RZ(params[0], wires=1)
+  
+  hilbert_template = qml.HilbertSchmidt(v_params, v_function=v_function, v_wires=[1], u_tape=u_tape)
```

Comment:
Maybe it's nice to have it in a QNode to see the integration too.

Reply:
Yes I change the example!